src/main/ruby/core/process.rb

```diff
@@ -129,6 +129,8 @@ def self.clock_gettime(id, unit=:float_second)
     case unit
     when :nanosecond
       time
+    when :microsecond
+      time / 1000
```

Comment:
I don't really care which format we use, but this one spells out `1000` and the `:float_microsecond` branch right below uses `1e3`. We should just just be consistent.

Reply:
it's integers versus floats (and we can't choose for integers), but we could write `1_000.0` and `1_000_000_000.0` for consistency.

Reply:
Oops. You're right.